include/beast/http/impl/basic_headers.ipp

```diff
@@ -205,6 +206,18 @@ basic_headers(FwdIt first, FwdIt last)
 }
 
 template<class Allocator>
+std::size_t
+basic_headers<Allocator>::
+count(boost::string_ref const& name) const
```

Comment:
Hmm, the combination of *list* and *set* reminds me of a certain coding
test...

On Tue, Jul 5, 2016 at 11:33 AM, Vinnie Falco <notifications@github.com>
wrote:

> In include/beast/http/impl/basic_headers.ipp
> <https://github.com/vinniefalco/Beast/pull/37#discussion_r69614540>:
>
> > @@ -205,6 +206,18 @@ basic_headers(FwdIt first, FwdIt last)
> >  }
> >
> >  template<class Allocator>
> > +std::size_t
> > +basic_headers<Allocator>::
> > +count(boost::string_ref const& name) const
>
> Whoops!!! Why use existing functions when you can write them again
> yourself!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/vinniefalco/Beast/pull/37/files/db58cf1c0f4f215c9bc15450e096e41e2c169748#r69614540>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADBpbhHFg-DlL482gKS3VusJJWL6pkcRks5qSqNjgaJpZM4JFX2l>
> .
>